### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by sebastian-lerner**
remove reference to really old image

# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

**Comment by sebastian-lerner** [comment]
@felicianotech can you double check this looks okay?

**Comment by sebastian-lerner** [comment]
https://circleci.atlassian.net/browse/CIRCLE-39102

File: jekyll/_cci2/faq.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ CircleCI does not infer from your project and is moving toward a model of smart
 {:.no_toc}
 Yes, you can use one of ours! For now, but this image may be deprecated in a future release.
 
-The `circleci/build-image:ubuntu-14.04-XL-922-9410082` image has the same content as the Ubuntu Trusty 14.04 image our web app uses. Just know that the image is fairly large (around 17.5 GB uncompressed), so it’s less ideal for local testing.
+See the [available machine images](https://circleci.com/docs/2.0/configuration-reference/#available-machine-images) section for the latest image to use.   These images have the same content as the image our web app uses. Just know that the image is fairly large (around 17.5 GB uncompressed), so it’s less ideal for local testing.
```
Comment (thomasr8) [inline jekyll/_cci2/faq.md:56]:
```suggestion
See the [available machine images](https://circleci.com/docs/2.0/configuration-reference/#available-machine-images) section for the latest image to use. These images have the same content as the image our web app uses. Just know that the image is fairly large (around 17.5 GB uncompressed), so it is less than ideal for local testing.
```

**Review by thomasr8 — APPROVED** [review]
Just a few very minor edits. Looks good!

**Review by felicianotech — APPROVED** [review]
Yup, this looks good to me.

**Comment by sebastian-lerner** [comment]
@thomasr8 could you merge when you get a chance? i don't have merge permissions

**Pull request merged**